### ISSUE TITLE
Can't open pdf Mojave

### ISSUE THREAD

**Issue by gonzales-jorge-siul** [zegervdv/homebrew-zathura#40]
I've installed Zathura + zathura-pdf-poppler following the instruccion and it works fine. Then when I tried to open a pdf it prompts: objc[123]: +[NSResponder initialize] may have been in progress in another thread when fork() was called.
I found a workaround at [link](https://stackoverflow.com/questions/50168647/multiprocessing-causes-python-to-crash-and-gives-an-error-may-have-been-in-progr): export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
But now it prompts:
**The process has forked and you cannot use this CoreFoundation functionality safely. You MUST exec().
Break on __THE_PROCESS_HAS_FORKED_AND_YOU_CANNOT_USE_THIS_COREFOUNDATION_FUNCTIONALITY___YOU_MUST_EXEC__() to debug**

I reinstalled Zathura but still dont work

The commands: 
  516  brew tap zegervdv/zathura
  517  brew install zathura
  518  brew reinstall zathura
  519  brew install zathura-pdf-poppler
  520  brew reinstall zathura-pdf-poppler
  521  vim ~/.config/zathura/zathurarc
  522  zathura --version
  523  mkdir -p $(brew --prefix zathura)/lib/zathura
  524  ln -s $(brew --prefix zathura-pdf-poppler)/libpdf-poppler.dylib $(brew --prefix zathura)/lib/zathura/libpdf-poppler.dylib
  525  zathura --version
  526  cd ./Downloads
  527  zathura - ./constancia.pdf 

MacBook Pro (13-inch, Mid 2012)
macOS Mojave 10.14.4

**Comment by zegervdv** [comment]
Weird. I’ll try to push the latest updates tomorrow. With a bit of luck it is solved already. I’ll update you once it is done.

**Comment by zegervdv** [comment]
can you try again with the latest updates?
I am stuck at High Sierra for now so cannot test.

**Comment by gonzales-jorge-siul** [comment]
Thank you! It works now.
First I tried reinstalling zathura and Zathura-pdf-poppler but it showed the same problem so I decided to update everything (brew and all the componentes) and uninstall Zathura (brew uninstall --force zathura) and Zathura-pdf-poppler. Then i reinstalled them and it worked. 
I'm pasting my history, I think it might be useful.
 brew install zathura
  487  brew upgrade zathura
  488  brew install zathura-pdf-poppler
  489  brew reinstall zathura-pdf-poppler
  490  brew reinstall zathura-pdf-poppler
  491  zathura --version
  492  mkdir -p $(brew --prefix zathura)/lib/zathura
  493  ln -s $(brew --prefix zathura-pdf-poppler)/libpdf-poppler.dylib $(brew --prefix zathura)/lib/zathura/libpdf-poppler.dylib
  494  zathura --version
  495  vim ~/.config/zathura/zathurarc
  496  cd Downloads/
  497  zathura - constancia.pdf 
  498  cd ../
  499  nano .bash_profile 
  500  vim .bash_profile 
  501  zathura - Downloads/constancia.pdf 
  502  brew update (from here )
  503  git status 
  504  brew update
  505  sudo brew update
  506  brew update
  507  brew update (up to this line I had problems with update)
  508  brew untap homebrew/homebrew-dupes ()
  509  brew update
  510  brew outdated
  511  brew upgrade dnsmasq
  512  brew upgrade libxml2
  513  brew outdated
  514  brew upgrade openssl
  515  brew outdated
  516  brew upgrade yarn 
  517  git outdated
  518  brew outdated
  519  brew list
  520  brew cleanup -n
  521  brew cleanup
  522  brew uninstall --force zathura-pdf-poppler
  523  brew style (here I cancelled this command with ctrl-c)
  524  brew uninstall --force zathura-pdf-poppler
  525  brew update-reset
  526  brew uninstall --force zathura-pdf-poppler
  527  brew uninstall --forcec zathura
  528  brew uninstall --force zathura
  529  brew list 
  530  brew install zathura
  531  brew install zathura-pdf-poppler
  532  mkdir -p $(brew --prefix zathura)/lib/zathura
  533  ln -s $(brew --prefix zathura-pdf-poppler)/libpdf-poppler.dylib $(brew --prefix zathura)/lib/zathura/libpdf-poppler.dylib
  534  zathura ./Downloads/constancia.pdf
